docs(content/mig-guide): migration to HeroDevs XLTS libraries (#1709)
camunda/camunda-docs-manual: content/update/patch-level.md

Please note that this does not apply to the Sync API, which keeps its behavior and fails if the process instance does not exist.

> ## 7.22.0 to 7.22.1 / 7.21.5 to 7.21.6 /  7.20.8 to 7.20.9

Author: We already had a section for 
```
## 7.22.0 to 7.22.1
```
I created a new section as this is relevant also for migrating: `7.21.5 to 7.21.6 /  7.20.8 to 7.20.9`
Let me know if there is a better way to do this.

Please note that this does not apply to the Sync API, which keeps its behavior and fails if the process instance does not exist.

### XLTS libraries provided by XLTS.dev by HeroDevs, Incl

> Starting with 7.22.1,  7.21.6, 7.20.9 the following libraries are replaced with XLTS versions provided by XLTS.dev by HeroDevs, Incl:

Reviewer: ```suggestion
These patches replace Bootstrap NES and AngularJS NES versions provided by HeroDevs, Inc.:
```

> *  *angular-ui-bootstrap*

Reviewer: ```suggestion
* *AngularJS* (technical names: `angular`, `angular-animate`, `angular-cookies`, `angular-loader`, `angular-mocks`, `angular-resource`, `angular-route`, `angular-sanitize`, `angular-touch`)
*  *angular-ui-bootstrap*
```

*  *angular-translate*
*  *angular-moment*

> *  *Bootstrap*

Reviewer: ```suggestion
*  *Bootstrap*

Where AngularJS, angular-ui-bootstrap, angular-translate, angular-moment, and Bootstrap were licensed entirely under the MIT license, Bootstrap NES and AngularJS NES by HeroDevs, Inc. licenses additional parts under the HeroDevs NES License. By downloading and using Camunda with Bootstrap NES and AngularJS NES by HeroDevs, Inc., you agree to the terms of the HeroDevs NES License. You can find the HeroDevs NES License terms in our [License Book](TODO: ADD LINK TO LICENSE BOOK).
```

> Please see our [third-Party libraries documentation]({{< ref "/introduction/third-party-libraries/_index.md#web-applications-cockpit-tasklist-admin" >}}) for details.

Reviewer: ```suggestion
Please see our [third-Party libraries documentation]({{< ref "/introduction/third-party-libraries/_index.md#web-applications-cockpit-tasklist-admin" >}}) for details.
```

Please note that this does not apply to the Sync API, which keeps its behavior and fails if the process instance does not exist.

### Bootstrap NES and AngularJS NES by HeroDevs, Inc.

> These patches replace Bootstrap NES and AngularJS NES versions provided by HeroDevs, Inc.:

Reviewer: ```suggestion
These patches replace the following libraries with versions of Bootstrap NES and AngularJS NES by HeroDevs, Inc.:
```
